cmd/pack/main.go

```diff
@@ -3,51 +3,65 @@ package main
 import (
 	"os"
 
+	"github.com/apex/log"
 	"github.com/buildpack/pack"
 	"github.com/buildpack/pack/commands"
 	"github.com/buildpack/pack/config"
+	clilogger "github.com/buildpack/pack/internal/logging"
 	"github.com/buildpack/pack/logging"
-
-	"github.com/fatih/color"
 	"github.com/spf13/cobra"
 )
 
 var (
-	Version           = "0.0.0"
-	timestamps, quiet bool
-	logger            logging.Logger
-	cfg               config.Config
-	packClient        pack.Client
+	Version    = "0.0.0"
+	cfg        config.Config
+	packClient pack.Client
 )
 
 func main() {
+	// create logger with defaults
+	handler := clilogger.NewLogHandler(os.Stderr)
+	logger := clilogger.NewLogWithWriter(handler)
+
 	cobra.EnableCommandSorting = false
 	rootCmd := &cobra.Command{
 		Use: "pack",
 		PersistentPreRun: func(cmd *cobra.Command, args []string) {
-			logger = *logging.NewLogger(os.Stdout, os.Stderr, !quiet, timestamps)
-			cfg = initConfig(&logger)
-			packClient = initClient(&cfg, &logger)
+			if fs := cmd.Flags(); fs != nil {
+				if flag, err := fs.GetBool("no-color"); err != nil {
```

Comment:
Binding to variables seems a bit simpler than calling `GetBool`, though this is pretty straightforward too. Is there a reason this way is preferred?